<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>连连看</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resource/yunian.ico</normaloff>resource/yunian.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">	font-family:微软雅黑;
	font-size:20px;
	color: rgb(156, 163, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: (255, 255, 255)</string>
    </property>
    <property name="text">
     <string>按ctrl + shift + Tab可停止 </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: (255, 255, 255)</string>
    </property>
    <property name="text">
     <string>鼠标点击速度:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>253</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>0.5秒</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="styleSheet">
        <string notr="true">	font-family:微软雅黑;
	font-size:20px;
	color:rgb(104, 132, 147);</string>
       </property>
       <property name="text">
        <string>0.05秒</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>无延迟</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: (255, 255, 255)</string>
    </property>
    <property name="text">
     <string>按住空格键屏蔽点击</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>311</width>
      <height>271</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>UpArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>dist/resource/background.jpeg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>112</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>按下此按钮,无倒计时时间</string>
    </property>
   </widget>
   <zorder>label_4</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label_3</zorder>
   <zorder>pushButton_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>282</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actiondd"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_1"/>
  </widget>
  <action name="action_1">
   <property name="icon">
    <iconset>
     <normaloff>resource/sign_out.jpeg</normaloff>resource/sign_out.jpeg</iconset>
   </property>
   <property name="text">
    <string>返回主界面</string>
   </property>
  </action>
  <action name="actiondd">
   <property name="text">
    <string>打开文档</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
